src/app/modules/map/components/map/map.component.ts

```diff
@@ -159,6 +159,8 @@ export class MapComponent implements OnInit, OnDestroy, AfterViewInit {
         }
       },
     });
+
+    this.isActive = new BehaviorSubject(this.autoActivate);
```

Comment:
Jeg vet ikke helt hva slags konsekvenser det har, men siden autoActivate er in Input, så kan det hende den ikke er gitt til komponenten enda når constructoren kjøres. Det er nok sånn at constructoren kjøres først, så settes inputene, så kjøres ngOnInit osv. Sjekk evt ut dette i angular-dokumentasjon og life cycle methods.